<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesWidget</class>
 <widget class="QWidget" name="PreferencesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>119</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of uploads</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="uploadsBox">
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Number of downloads</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="downloadsBox">
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Download directory</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="downloadDirEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="downloadDirButton">
         <property name="text">
          <string>Choose...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>119</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>uploadsBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PreferencesWidget</receiver>
   <slot>emitNumberOfUploadsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadsBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PreferencesWidget</receiver>
   <slot>emitNumberOfDownloadsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadDirEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PreferencesWidget</receiver>
   <slot>emitDownloadDirChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>emitNumberOfDownloadsChanged(int)</slot>
  <slot>emitNumberOfUploadsChanged(int)</slot>
  <slot>emitDownloadDirChanged(QString)</slot>
 </slots>
</ui>
